src/centos/scripts/start-marklogic.sh

```diff
@@ -96,6 +98,61 @@ if [ -n "${TZ}" ]; then
     echo "${TZ}" | sudo tee /etc/timezone
 fi
 
+################################################################
+# restart_check(hostname, baseline_timestamp, caller_lineno)
+#
+# Use the timestamp service to detect a server restart, given a
+# a baseline timestamp. Use N_RETRY and RETRY_INTERVAL to tune
+# the test length. Include authentication in the curl command
+# so the function works whether or not security is initialized.
+#   $1 :  The hostname to test against
+#   $2 :  The baseline timestamp
+#   $3 :  Invokers LINENO, for improved error reporting
```

Comment:
We are not using third parameter $3 anywhere. Description says for improved error reporting, but couldn't find where it is used in restart_check()

Reply:
I copied that directly from the docs, comment included, we are not using it and that comment should be removed